pipeline_products_session/jwst-data-products-part2.ipynb

```diff
@@ -0,0 +1,1401 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's start with the imaging example to interact with the WCS."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After the Assign WCS step is run in Stage 2, a GWCS object that contains all the transforms is now attached to the image model. What does the GWCS WCS look like? "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cal_image.meta.wcs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "How does this compare to the FITS WCS standard?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_fits_wcs = cal_image.get_fits_wcs()\n",
+    "image_fits_wcs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compare that with the WCS information in the datamodel:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('CRVAL: ', cal_image.meta.wcsinfo.crval1, cal_image.meta.wcsinfo.crval2)\n",
+    "print('CRPIX: ', cal_image.meta.wcsinfo.crpix1, cal_image.meta.wcsinfo.crpix2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can see what coordinate frames are available to you:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cal_image.meta.wcs.available_frames  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cal_image.meta.wcs.input_frame"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cal_image.meta.wcs.output_frame"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can check the transform from detector pixels to sky coordinates in decimal degrees of RA and Dec: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cal_image.meta.wcs(110, 110)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "detector_to_world = cal_image.meta.wcs.get_transform('detector', 'world')"
```

Comment:
I think this whole WCS section is really useful. One small suggestion would be to show that the transformations can run on lists of pixels, i.e. it doesn't have to run on one pixel at a time.

Reply:
Good idea, added an example